src/index.js

```diff
@@ -14,41 +14,54 @@ function layer(context, selectedLayer) {
   };
 }
 
-function styleguideColors(context, colors) {
+function textStyles(context) {
+  var code = "// MARK: - Project Texts styles \n\n";
+  
+  var textStylesProject = context.project.textStyles
+  var textStylesAll = textStylesProject.concat(context.project.linkedStyleguide.textStyles)
 
-  var code = "// MARK: - Project Color palette\n\n";
-  code += swiftColors.getColorsSwiftSnippet(context, colors, false);
+  code += swiftTextStyles.getTextStylesSwiftSnippet(context, textStylesAll, false)
```

Comment:
Your code to fetch text styles (also works for colors) is duplicated for snippets and for export. Can you make a private function in swiftTextStyles.js to concat all textStyles together so that we no longer need to pass them in index.js ? It has the advantage of removing a parameter and keeping index.js at the minimum.